Euler1PS_v2.js

```diff
@@ -0,0 +1,25 @@
+//https://projecteuler.net/problem=1
+
+
+
+function findTheSum(){
+    let input = prompt("Input a number:");

```

Comment:
You said in your commit message:

```
I'm not sure how I would write tests for this other than to take numbers I can work and make sure they product the desired result.
```

That's *exactly* what you should do. For example, you know that if the number is 1, then the sum should be 0. So, you might do something like:

```javascript
let tests = [
    { "input": 1, "expected: 0 },
    { "input": 3, "expected: 3 },
    { "input": 5, "expected: 8 },
    { "input": 6, "expected: 14 },
];

tests.forEach(testcase => {
    let output = find_the_sum(testcase.input);
    if (output == testcase.output) {
        console.log("Input: " + testcase.input + " - Ok");
    }
    else {
        console.log("Input: " + testcase.input + " - NOT Ok");
    }
});
```